packages/cli/src/public-api/v1/handlers/executions/spec/schemas/execution.yml

```
@@ -24,6 +24,7 @@ properties:
   stoppedAt:
     type: string
     format: date-time
+    nullable: true
```

To help consumers of the API understand the impact of this `nullable` better, we can add a description to the property.

```suggestion
    nullable: true
    description: The time at which the execution stopped. Will only be null for executions that still have the status 'running'.
```